src/ctype/CType.spec.ts

```diff
@@ -63,7 +63,7 @@ describe('CType', () => {
       owner: identityAlice.address,
     }
 
-    const resultTxStatus = new TxStatus(FINALIZED, Crypto.hashStr('987654'))
```

Comment:
This is out of the scope of the ticket.

Reply:
A change will be necessary though if we’re changing the type of TxStatus.payload to object. We could pass an object instead, but this is dead code anyway, it is never used afterwards.